sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -36,8 +36,7 @@
                 />
                 <field
                     name="order_partner_id"
-                    domain="[('customer','=',True)]"
-                    context="{'search_default_customer':1, 'show_address': 1}"
+                    context="{'show_address': 1}"
```

Comment:
Please pass also the same context as in v13 regular sale order view that makes the many2one to search first partners with customer rank.